superset/config.py

```diff
@@ -466,7 +466,6 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     "EMBEDDABLE_CHARTS": True,
     "DRILL_TO_DETAIL": False,
     "DATAPANEL_CLOSED_BY_DEFAULT": False,
-    "CROSS_REFERENCES": False,
```

Comment:
Is the preferred strategy to first enable the `CROSS_REFERENCES` feature by default and then deprecate the feature in version 3.0? 

Also [SIP-57](https://github.com/apache/superset/issues/12566) changing the default or removing a feature flag is deemed a breaking change and thus the appropriate actions need to be undertaken.

Reply:
This one is a little funny, because the feature flag never went out in a version, and will not. So adding and removing it between published versions was perceived as a non-issue. We were hoping to implement a good means of launching darkly using this sort of ephemeral feature flag in lieu of a giant feature branch. Perhaps if we need to do this again, we can add a new category of "Pending release" or something to FEATURE_FLAGS.md to explain these?

I'll bring this up in Town Hall, and be seeking more opinions in general, but hopefully we can let this one through since the FF was never released or intended to be released.